src/main/java/core/basesyntax/handler/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class BalanceOperationHandler implements OperationHandler {
```

Comment:
According to the checklist, there shouldn't be an empty line at the beginning of a class or method implementation. Please remove the redundant empty line after the package statement.